### ISSUE TITLE
Custom class functions

### ISSUE THREAD

**Issue by quiin** [emehrkay/Pypher#33]
Is there a way to create custom functions by subclassing `Func` or `FuncRaw`?
I noticed on the readme there's a section about it but I am not able to reproduce it. 
This is may attempt

```python
# Define de function
from pypher.builder import Func
class ApocIterate(Func):
  _CAPITALIZE = False
  _ALIASES = ['periodic_iterate', 'apoc_periodic_iterate']
  name = 'apoc.periodic.iterate'

# Attempt to use it
from pypher import Pypher
q = Pypher()
q.apoc_iterate

# Throws the following error
/usr/local/lib/python3.7/site-packages/pypher/builder.py in __getattr__(self, attr)
    302             link = Property(name=attr.strip('__'))
    303         elif attr_low in _LINKS:
--> 304             link = (getattr(_MODULE, _LINKS[attr_low]))()
    305         else:
    306             link = Statement(name=attr)

AttributeError: module 'pypher.builder' has no attribute 'ApocIterate'
```

I know we can define custom functions using `create_function`; but I'd like to define them as classes. Is this indeed supported?

**Comment by emehrkay** [comment]
This was an error with line 304. It assumes that all class `Func` and `Statement` objects are apart of the pypher.builder module. 

If you pull an update from here for pypi, it should be fixed -- version 0.17.1 

Tester:

![Screen Shot 2019-11-26 at 6 34 22 PM](https://user-images.githubusercontent.com/24242/69681271-91b80700-107b-11ea-92fc-7be9e547ed84.png)


Good catch, thanks

**Comment by quiin** [comment]
Awesome, works like a charm!
Thanks for addressing it so quickly.
